config.go

```diff
@@ -0,0 +1,42 @@
+package logs
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	defaultSTAGE = "unknown"
+)
+
+type StackTraceConfig struct {
+	Enable  bool
+	Context int `yaml:"context"`
+}
+
+type SentryConfig struct {
+	Enable          bool             `yaml:"enable"`
+	Stage           string           `yaml:"stage"`
+	MinlLogLevel    string           `yaml:"min_log_level"`
```

Comment:
maybe use just "level" key for the most compatibility with old services?

Reply:
1. log level info.
2. sentry min log level error.

StdOut logs will fires from 1.
Sentry will logs from 2.